### ISSUE TITLE
Automatically set update_ocn_f = .true. when coupling with MOM

### ISSUE THREAD

**Issue by gustavo-marques** [ESCOMP/CESM_CICE5#44]
Is it possible to automatically set `update_ocn_f = .true.` when coupling with MOM? 
I thought this had already being done but in my latest cases with cesm2_3_alpha05c, `update_ocn_f = .false.` by default. 
pinging @alperaltuntas and @dabail10

**Comment by dabail10** [comment]
Not sure what happened here. I don't have a record of this changing in namelist_defaults_cice.xml. Was this in the MOM scripts? I know we talked about keying off the grid, but I felt like this was not the best way. Ideally, it should key off comp_ocn = mom6 or whatever. I thought we lumped this in with tfrz_option?

**Comment by alperaltuntas** [comment]
I just sent a PR that fixes this issue. As for `tfrz_option`, are we happy with `mushy` for mom6 coupling, or should it be compset dependent?

**Comment by gustavo-marques** [comment]
@alperaltuntas, thank you very much. I do not know what should be done for `tfrz_option` so I will let @dabail10 answer that.

**Comment by dabail10** [comment]
I'm pretty sure you do not have tfrz_option = 'mushy' in MOM. So, I think we had to do linear_salt. We get this from the mediator:

    call NUOPC_CompAttributeGet(gcomp, name="tfreeze_option", value=tfrz_option, isPresent=isPresent, rc=rc)
    if (ChkErr(rc,__LINE__,u_FILE_u)) return
    if (.not. isPresent) then
       tfrz_option = 'linear_salt'  ! TODO: is this right? This must be the same as mom is using for the calculation.
    end if

**Comment by gustavo-marques** [comment]
I can confirm that `TFREEZE_SALTWATER_OPTION = linear_salt` in my out-of-the-box MOM cases. This is a CIME variable. To clarify, does that overwrites  `tfrz_option` in CICE?

**Comment by dabail10** [comment]
Yes. It is done in the cap. I thought update_ocn_f was also overwritten in the cap, but it does not appear to be. I think this is the way we had wanted to go. Nonetheless, give me a few weeks. I will be switching CICE6 and CICE5 as the defaults at which point I will make sure that update_ocn_f is set correctly.
